### DISCUSSION TITLE
Seeing Starred Repos not already on a List[s]?

### DISCUSSION THREAD

**Discussion by dreamingbinary** [community/community#28515] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to know what I haven't put on a list yet. Any easy way to fetch/query against this data?

**Comment by radar-blip** [comment]
I dont think they have thought that far ahead yet.

**Comment by k3rielit** [comment, marked as answer]
I've quickly thrown together a script for this, in case you need it: [repo](https://github.com/k3rielit/scripts/tree/main/github)

Just paste it into the console, change the `uid` on the first line to your name, press enter, wait for it to complete, then run commands (a list of them is in the readme).

It'll also print every listed/starred repo in categories onto the page as links.

**Reply by boscacci** [reply]
Thanks for this workaround.

**Comment by boscacci** [comment]
As a compulsive organizer, I would also enjoy the ability to see which of my starred repos have not yet made it onto any list.
